flang/runtime/time-intrinsic.cpp

```diff
@@ -19,6 +19,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <ctime>
+#include <chrono>
```

Comment:
Is this include still needed?

Reply:
Nope. It can be removed.  I think the code can be greatly simplified.

If there's consensus amongst the reviewers that the way I did things are fine, I'll refactor the code and remove everything that's no longer needed.